### ISSUE TITLE
Validation stalls when Geometry is empty

### ISSUE THREAD

**Issue by edigonzales** [claeis/ilivalidator#216]
```
<OeREBKRMtrsfr_V1_1.Transferstruktur.Geometrie TID="1040">
  <Flaeche_LV95>
    <SURFACE></SURFACE>
  </Flaeche_LV95>
<Rechtsstatus>inKraft</Rechtsstatus>
<publiziertAb>2014-04-22</publiziertAb>
<Eigentumsbeschraenkung REF="48"></Eigentumsbeschraenkung>
<ZustaendigeStelle REF="ch.2403"></ZustaendigeStelle></OeREBKRMtrsfr_V1_1.Transferstruktur.Geometrie>
```

ilivalidator just validates eternally:

```
Info: ilifile </Users/stefan/.ilicache/models.geo.admin.ch/V_D/OeREB/OeREBKRMtrsfr_V1_1.ili>
Info: validate data...
Info: assume unknown external objects
Info: first validation pass...
```

**Comment by claeis** [comment]
see #322

**Comment by claeis** [comment]
fixed in c3b99887c693ae449ff85d0af94ecb6f736efa59
